src/governance.ts

```diff
@@ -128,10 +133,51 @@ export class GovernanceCanister {
     return this.responseConverters.toListProposalsResponse(rawResponse);
   };
 
+  public stakeNeuron = async ({
+    stake,
+    principal,
+    ledgerCanister,
+  }: {
+    stake: ICP;
+    principal: Principal;
+    ledgerCanister: LedgerCanister;
+  }): Promise<void> => {
```

Comment:
I am surprised this method is returning `void`. Shouldn't we be returning the `NeuronId` that was created?

Reply:
I can return that, it makes sense. I am still not sure how we want to do this in our frontend. But returning the `NeuronId` is a good assumption.